src/feedback.js

```diff
@@ -13,6 +13,7 @@ export default class Feedback {
 
 		const defaultOptions = {
 			id: 'feedback',
+			emailplaceholder: 'Email (optional)',
```

Comment:
Also, please move the option below the `inputPlaceholder` option to make the code a bit cleaner.